js-oop/game/game.js

```diff
@@ -0,0 +1,21 @@
+function GameObject(imageUrl) {
+    this.imageUrl = imageUrl;
+    this.ready = false;
+    this.x = 0;
+    this.y = 0;
+}
+
+GameObject.prototype.image = new Image();
+GameObject.prototype.onload = function () {
+    this.ready = true;
+}
+
+const gameObj = new GameObject('http://www.kenart.net/portfolio/images/gameprojects_alice.jpg');
+console.log(gameObj);
+gameObj.onload();
+gameObj.image.src = gameObj.imageUrl;
```

Comment:
image.src must be initialized in constructor and only in case if imageUrl was passed to constructor.